Back-end/Repository/PostRepository.cs

```diff
@@ -197,5 +201,38 @@ public double getPostRating(MySqlConnection connection, int id)
             }
             return rating;
         }
+
+        public List<Post> getPostsByGenreID(MySqlConnection connection, int genreId)
```

Comment:
Do MySqlConnection have base class? Can we use polymorphism here?

Reply:
interesting idea